AppConnectSwift/MultiPageFormViewController.swift

```diff
@@ -7,12 +7,12 @@ class MultiPageFormViewController: UIViewController, UIPageViewControllerDelegat
     @IBOutlet var previousButton : UIButton!
     @IBOutlet var nextButton : UIButton!
     
-    private var form : MDForm!
-    private var stepSequencer : MDStepSequencer!
+    fileprivate var form : MDForm!
```

Comment:
Nitpick: This file uses spaces before the `:`, which is a bit odd. I only use it in dictionary declarations.